packages/yuujinchou/yuujinchou.0.9/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "Generic name manipulation combinators"
+description: """
+This library implements a generic library for selecting names. It intends to facilitate the implementation of the "import" statement or any feature that allows users to select a group of names by patterns.
+"""
+maintainer: "favonia <favonia@gmail.com>"
+authors: "favonia <favonia@gmail.com>"
+license: "Apache 2.0"
+homepage: "https://github.com/favonia/yuujinchou"
+bug-reports: "https://github.com/favonia/yuujinchou/issues"
+dev-repo: "git+https://github.com/favonia/yuujinchou.git"
+depends: [
+  "dune" {build}
+  "ppx_deriving" {>= "4.5" & < "4.6"}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest"]
```

Comment:
this needs to be annotated with `{with-test}` so that it's only run when tests are requested.

Reply:
Thank you. Fixed.